app/models/queue_item.rb

```diff
@@ -0,0 +1,17 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :video
+  belongs_to :user
+  belongs_to :review
```

Comment:
QueueItem does not belong to a review. 1 QueueItem represents 1 Video and by fetching an associated video, you access the reviews.